backend/pkg/modules/internal/fuzzer/tools/secret.go

```diff
@@ -0,0 +1,88 @@
+// Copyright © 2022 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tools
+
+import (
+	"context"
+
+	uuid "github.com/satori/go.uuid"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	ImagePullSecretNamePrefix = "imagepullsecret"
+	SecretType                = "Opaque"
+	SecretKey                 = "FuzzerAuthData" // nolint:gosec
```

Comment:
`SecretKey` and `ImagePullSecretNamePrefix` - do you need the const to be exportable?

Reply:
No.
I will set it no exportable.